listenbrainz_spark/stats_writer/stats_writer.py

```diff
@@ -30,7 +30,7 @@ def connect_to_rabbitmq(self):
                 )
                 self.connection =  pika.BlockingConnection(connection_parameters)
                 break
-            except Exception as err:
+            except:
```

Comment:
this will error out because `err` is used in the except block. Why did you change this?

Reply:
It is more pythonic to be specific, so I'd prefer to go back to except Exception as err.